pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -434,6 +434,11 @@ func newKubeVirtConfigForCR(cr *hcov1alpha1.HyperConverged, namespace string) *c
 			"selinuxLauncherType": "spc_t",
 		},
 	}
+	val, ok := os.LookupEnv("SMBIOS")
```

Comment:
Maybe it makes sense to look for all the list of env variables here, for example machine type and smbios both

Reply:
currently we are not setting machine type in kubevirt-config, how is it named?

Reply:
found: https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-config/config-map.go#L49

Reply:
done